src/components/NewsItem/NewsItem.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import NewsItem from './index';
+
+const testProps = {
+  description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore...',
+  date: '1 day ago',
+}
+
+storiesOf('NewsItem', module).add('normal', () => (
+  <div>
+    <NewsItem title="Normal" featured={false} {...testProps}/>
+  </div>
+));
+
+storiesOf('NewsItem', module).add('featured', () => (
```

Comment:
No need to keep making `storiesOf`.

Move `bestivalProps` to the top, just below `testProps`. And then you can chain as main `.add`s as you want.

```js
storiesOf('NewsItem')
  .add('normal', stuff)
  .add('bestival-test', stuff)
  .add('another one', stuff);
```